### ISSUE TITLE
What the hell is going on with releases

### ISSUE THREAD

**Issue by AdamWill** [jhthorsen/mojolicious-plugin-assetpack#131]
Uh...yeah?

This repo has a commit that claims to be a 2.00 release, but it isn't tagged or actually released here or on CPAN, AFAICT. Then after that, there was a 1.48 release.

Meanwhile on CPAN, there is a 2.01 release that was made by someone else, Marcus Ramberg:

http://search.cpan.org/~mramberg/Mojolicious-Plugin-AssetPack-2.01/

which has a link to a different github repo, which doesn't seem to exist:

https://github.com/marcus/mojolicious-plugin-assetpack

so, uh, what the heck is going on here? Fedora has packaged this '2.01 release', but I am extremely unsure whether it *should* have done. It'd be nice to have some clarity here. thanks.

@marcus

**Comment by jhthorsen** [comment]
I think it was right before I went on vacation that I messed up big time. I had a change where I wanted to remove the use of assetpack.db, but that wasn't done so I had to remove those changes. Unfortunately, I had marked the "2.00" release in the Changes file in a commit that I forgot to revert, so what was supposed to be 1.48 ended up as 2.00.

I asked *people* if it was possible to revert the 2.00 release, and after some ideas I tried to remove it and go back to 1.48. This in the end ended up to be a very bad idea, so I got @marcusramberg to make a new release for me (2.01) while I was on vacation.

I really wish I could go back to 1.48, but it does not seem to be possible.
